pom.xml

```diff
@@ -124,6 +124,49 @@
           </execution>
         </executions>
       </plugin>
+      <!-- Spotless plugin: keeps the code formatted following the google-java-styleguide -->
+      <plugin>
+      <groupId>com.diffplug.spotless</groupId>
+      <artifactId>spotless-maven-plugin</artifactId>
+      <version>2.40.0</version>
+      <executions>
+        <execution>
+          <goals>
+            <goal>check</goal>
+          </goals>
+        </execution>
+      </executions>
+      <configuration>
+      <formats>
+        <format>
+          <includes>
+            <include>*.md</include>
+            <include>.gitignore</include>
+          </includes>
+          <trimTrailingWhitespace/>
+          <endWithNewline/>
+          <indent>
+            <spaces>true</spaces>
+            <spacesPerTab>4</spacesPerTab>
+          </indent>
+        </format>
```

Comment:
I kept the `<format>` section, let me know if you prefer to remove it (or remove the `.md`)